feeds/wlan-ap/opensync/src/platform/openwrt/src/netifd/src/wifi_inet_config.c

```diff
@@ -255,11 +256,19 @@ static int wifi_inet_conf_add(struct schema_Wifi_Inet_Config *iconf)
 	} else
 		blobmsg_add_bool(&del, "type", 1);
 
-	if (iconf->parent_ifname_exists && iconf->vlan_id > 2 && !strcmp(iconf->if_type, "vlan")) {
+	if (iconf->parent_ifname_exists && iconf->vlan_id > 2) {
 		char uci_ifname[256];
+		if(!strncmp(iconf->parent_ifname, "gre", strlen("gre"))) {
+			snprintf(uci_ifname, sizeof(uci_ifname), "gre4t-%s.%d", iconf->parent_ifname, iconf->vlan_id);
+			blobmsg_add_string(&b, "ifname", uci_ifname);
+			blobmsg_add_string(&b, "type", "bridge");
+		}
+		else {
+			LOG(INFO, "Creating WAN/LAN bridge");
```

Comment:
If we need this log @ line 267, should there be equivalent at 261?  Possibly neither required.

Reply:
Yes. Not required. Kind of missed it. Will get rid of it.